internal/serve/httphandler/receiver_handler_test.go

```diff
@@ -1616,3 +1616,42 @@ func Test_ReceiverHandler_BuildReceiversResponse(t *testing.T) {
 	err = dbTx.Commit()
 	require.NoError(t, err)
 }
+
+func Test_ReceiverHandler_GetReceiverVerificatioTypes(t *testing.T) {
+	dbt := dbtest.Open(t)
+	defer dbt.Close()
+
+	dbConnectionPool, err := db.OpenDBConnectionPool(dbt.DSN)
+	require.NoError(t, err)
+	defer dbConnectionPool.Close()
+
+	models, err := data.NewModels(dbConnectionPool)
+	require.NoError(t, err)
```

Comment:
Why is this necessary in this test? It isn't being used on the handler's function you're testing. Please only use the necessary for the test.

Reply:
I need `dbConnectionPool` and `models` to set the `handler`, isn't it?